services/ml/compilation_delegate_mps.mm

```diff
@@ -770,4 +769,22 @@ void ComputeMPSOffsetForImplictPadding(bool same_padding,
   return true;
 }
 
+bool CompilationDelegateMPS::CompileArgmax(
+    std::map<uint32_t, MPSNNImageNode*>& image_nodes,
+    const mojom::OperationPtr& operation) {
+  DLOG(INFO) << "Compile argmax operation.";
+
+  if (@available(macOS 10.14.1, *)) {
+    MPSNNReductionFeatureChannelsArgumentMaxNode* argmax_node =
```

Comment:
If MPS only supports reduction on feature channel, we may need to report error for other axis.

Reply:
Thanks, Done.